File: Week_2/OOPS/src/main/java/app/music/string/Veena.java
package music.string;

import music.Playable;

public class Veena implements Playable {
    String args;

    public Veena(String args) {
        this.args = args;
    }

    @Override
    public void play() {

        System.out.println(this.args);
    }
}

File: Week_2/OOPS_RUST/src/music/string/veena.rs
use crate::music::playable::Playable;

pub struct Veena {
    args: String,
}

impl Veena {
    pub fn new(args: String) -> Self {
        Self { args }
    }
}

impl Playable for Veena {
    fn play(&self) {
        println!("{}", self.args);
    }
}
